charts/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 6.24.1
+version: 6.25.1
```

Comment:
Thanks for the contribution, @chlumper! 🥇 

Could you please fix the version so that it is equal to `6.25.0`?

Reply:
I changed the version to `6.25.0` and I also added some missing doc for the values in the readme.